genai-perf/genai_perf/record/types/gpu_power_usage.py

```diff
@@ -20,86 +20,33 @@
 @total_ordering
 class GPUPowerUsage(DecreasingGPURecord):
     """
-    GPU Power Usage
+    GPU's power usage metric
     """
 
     tag = "gpu_power_usage"
 
     def __init__(self, value, device_uuid=None, timestamp=0):
-        """
```

Comment:
Same question as above: why not keep the docstring?

Reply:
Same answer :-).